app/models/repository_host/bitbucket.rb

```diff
@@ -70,6 +70,10 @@ def download_pull_requests(token = nil)
       nil
     end
 
+    def retrieve_commits(token = nil)
+      api_client(token).repos.commits.list(repository.owner_name, repository.project_name)['values']
```

Comment:
How doe sthis work with rate limiting?

Reply:
This is the only documentation I can find on rate limits: https://confluence.atlassian.com/bitbucket/rate-limits-668173227.html?_ga=2.192562154.1743157817.1561468931-1728484474.1527255352. Looks to be 1000 requests/hour on the repositories and commits. It looks like the rate is tied to the IP address if unauthenticated and the user if it is authenticated.